incubator/openvpn/values.yaml

```diff
@@ -47,9 +46,22 @@ persistence:
   accessMode: ReadWriteMany
   size: 1Gi
 
+easyrsa:
+  vars:
+    EASYRSA_REQ_COUNTRY: "US"
+    EASYRSA_REQ_PROVINCE: "California"
+    EASYRSA_REQ_CITY: "San Francisco"
+    EASYRSA_REQ_ORG: "Copyleft Certificate Co"
+    EASYRSA_REQ_EMAIL: "me@example.net"
+    EASYRSA_REQ_OU: "My Organizational Unit"
+    EASYRSA_CA_EXPIRE: "3650"
+    EASYRSA_CERT_EXPIRE: "3650"
+    EASYRSA_CRL_DAYS: "180"
```

Comment:
I think we should just move `OVPN_RENEG_SEC` here. We can default it to 0. No need to maintain it separately. You can also remove it from the "special" section at the top of `values.yaml`